### ISSUE TITLE
The filtering has been running for 3 days now, no warning is given but something must be broken

### ISSUE THREAD

**Issue by Kdoganyi** [sandberg-lab/Smart-seq3#8]
Hi there, 
I am trying to analyze my miseq data (smartseq3 method was used). 

After opening a virtual env,  I started a screen and ran the command below: 

**zUMIs/zUMIs.sh -y /public/groups/hausslerlab/people/kivilcim/iMGL/yamls/KDiMGL1st_sp.yaml 
Warning messages:
1: In if (!file.exists(x$name)) { :
  the condition has length > 1 and only the first element will be used
2: In if (!file.exists(x$name)) { :
  the condition has length > 1 and only the first element will be used
3: In if (!file.exists(x$name)) { :
  the condition has length > 1 and only the first element will be used
4: In if (!file.exists(x$name)) { :
  the condition has length > 1 and only the first element will be used

 You provided these parameters:
 YAML file:	/public/groups/hausslerlab/people/kivilcim/iMGL/yamls/KDiMGL1st_sp.yaml
 zUMIs directory:		/public/groups/hausslerlab/people/kivilcim/iMGL/zUMIs
 STAR executable		/public/groups/hausslerlab/people/sseiler/Programs/STAR/STAR-2.7.4a/bin/Linux_x86_64_static/STAR
 samtools executable		samtools
 pigz executable		pigz
 Rscript executable		Rscript
 RAM limit:   null
 zUMIs version 2.9.7 

Mon Mar  3 15:14:54 PST 2022
Filtering...
stat: missing operand**

Even though I detached the screen, whenever I went back to the screen program appeared to be running. 

My understanding is that generating one big yaml file for everything is supposed to work the best. 
I am attaching my yaml file:
[KDiMGL1st_sp.yaml.txt](https://github.com/sandberg-lab/Smart-seq3/files/8201595/KDiMGL1st_sp.yaml.txt)

I have tried doing this so many ways, and whenever I think I fixed something I run into a different problem. However, this time I actually have no idea what is going on. Why am I not getting any warning but it feels like something is broken? Am I in an infinite loop somehow? 

Thank you so much already!

**Comment by cziegenhain** [comment]
Hi,

You can only give one filepath under the 'name' field.
Seems like your data is already demultiplexed, you can use the non-demultiplexed data (if you dont have it you could concatenate the fastq files from many cells).

With the typical output of a MiSeq run, I would think the Filtering step  should only take a few minutes!
Best,
Christoph

**Comment by Kdoganyi** [comment]
Thanks a lot for your response! I think that worked!

**Comment by Kdoganyi** [comment]
Hi there! I have a new issue that I just started getting yesterday.  Should I end this one and start a new repository? Basically, after getting some analysis done, and getting some data out, I stopped making any process. The issue is that now when I run the program (zUMIs/zUMIs.sh -y <yaml> location)   the program keeps halting at the first stage, saying there was a parse error and from there on I am getting empty files.  Below I pasted what I get as a warning
"
[W::sam_read1] Parse error at line 3
[main_samview] truncated file.
[W::sam_read1] Parse error at line 3
[main_samview] truncated file.
[W::sam_read1] Parse error at line 3
[main_samview] truncated file.
[W::sam_read1] Parse error at line 3
[main_samview] truncated file.
cat: /public/groups/hausslerlab/people/kivilcim/iMGL/outputall/zUMIs_output/.tmpMerge//KDiMGLall.*.BCstats.txt: No such file or directory
Mon Mar 14 20:24:07 PDT 2022
Error in eval(bysub, parent.frame(), parent.frame()) : 
  object 'XC' not found
Calls: cellBC -> [ -> [.data.table -> eval -> eval
In addition: Warning message:
In data.table::fread(bccount_file, header = FALSE, col.names = c("XC",  :
  File '/public/groups/hausslerlab/people/kivilcim/iMGL/outputall/KDiMGLall.BCstats.txt' has size 0. Returning a NULL data.table.
Execution halted
"
This did not happen when I was using the same data 2 days ago, and  my yaml file appears to be formatted correctly.  I have seen some comments about this issue in the zUMIs repository. Let me know, if you believe it is a better idea to post here, at zUMIs repository, or if I should start a new issue. Depending on your answer I can include all my files,  links to what I have tried and explain the situation better. Truly appreciative of your help! Thank you so much!

**Comment by Kdoganyi** [comment]
Hi there! I have a new issue that I just started getting yesterday.  Should I end this one and start a new repository? Basically, after getting some analysis done, and getting some data out, I stopped making any process. The issue is that now when I run the program (zUMIs/zUMIs.sh -y <yaml> location)   the program keeps halting at the first stage, saying there was a parse error and from there on I am getting empty files.  Below I pasted what I get as a warning
"
[W::sam_read1] Parse error at line 3
[main_samview] truncated file.
[W::sam_read1] Parse error at line 3
[main_samview] truncated file.
[W::sam_read1] Parse error at line 3
[main_samview] truncated file.
[W::sam_read1] Parse error at line 3
[main_samview] truncated file.
cat: /public/groups/hausslerlab/people/kivilcim/iMGL/outputall/zUMIs_output/.tmpMerge//KDiMGLall.*.BCstats.txt: No such file or directory
Mon Mar 14 20:24:07 PDT 2022
Error in eval(bysub, parent.frame(), parent.frame()) : 
  object 'XC' not found
Calls: cellBC -> [ -> [.data.table -> eval -> eval
In addition: Warning message:
In data.table::fread(bccount_file, header = FALSE, col.names = c("XC",  :
  File '/public/groups/hausslerlab/people/kivilcim/iMGL/outputall/KDiMGLall.BCstats.txt' has size 0. Returning a NULL data.table.
Execution halted
"
This did not happen when I was using the same data 2 days ago, and  my yaml file appears to be formatted correctly.  I have seen some comments about this issue in the zUMIs repository. Let me know if you believe it is a better idea to post it there, here or if I should start a new issue. Depending on your answer I can include all my files,  links to what I have tried and explain the situation better. Truly appreciative of your help! Thank you so much!

**Comment by Kdoganyi** [comment]
Hi there! I have a new issue that I just started getting yesterday.  Should I end this one and start a new repository? Basically, after getting some analysis done, and getting some data out, I stopped making any process. The issue is that now when I run the program (zUMIs/zUMIs.sh -y <yaml> location)   the program keeps halting at the first stage, saying there was a parse error and from there on I am getting empty files.  Below I pasted what I get as a warning
"
[W::sam_read1] Parse error at line 3
[main_samview] truncated file.
[W::sam_read1] Parse error at line 3
[main_samview] truncated file.
[W::sam_read1] Parse error at line 3
[main_samview] truncated file.
[W::sam_read1] Parse error at line 3
[main_samview] truncated file.
cat: /public/groups/hausslerlab/people/kivilcim/iMGL/outputall/zUMIs_output/.tmpMerge//KDiMGLall.*.BCstats.txt: No such file or directory
Mon Mar 14 20:24:07 PDT 2022
Error in eval(bysub, parent.frame(), parent.frame()) : 
  object 'XC' not found
Calls: cellBC -> [ -> [.data.table -> eval -> eval
In addition: Warning message:
In data.table::fread(bccount_file, header = FALSE, col.names = c("XC",  :
  File '/public/groups/hausslerlab/people/kivilcim/iMGL/outputall/KDiMGLall.BCstats.txt' has size 0. Returning a NULL data.table.
Execution halted
"
This did not happen when I was using the same data 2 days ago, and  my yaml file appears to be formatted correctly.  I have seen some comments about this issue in the zUMIs repository. Let me know if you believe it is a better idea to post it there, here or if I should start a new issue. Depending on your answer I can include all my files,  links to what I have tried and explain the situation better. Truly appreciative of your help! Thank you so much!

**Comment by Kdoganyi** [comment]
Hi there! I have a new issue that I just started getting yesterday.  Should I end this one and start a new repository? Basically, after getting some analysis done, and getting some data out, I stopped making any process. The issue is that now when I run the program (zUMIs/zUMIs.sh -y <yaml> location)   the program keeps halting at the first stage, saying there was a parse error and from there on I am getting empty files.  Below I pasted what I get as a warning
"
[W::sam_read1] Parse error at line 3
[main_samview] truncated file.
[W::sam_read1] Parse error at line 3
[main_samview] truncated file.
[W::sam_read1] Parse error at line 3
[main_samview] truncated file.
[W::sam_read1] Parse error at line 3
[main_samview] truncated file.
cat: /public/groups/hausslerlab/people/kivilcim/iMGL/outputall/zUMIs_output/.tmpMerge//KDiMGLall.*.BCstats.txt: No such file or directory
Mon Mar 14 20:24:07 PDT 2022
Error in eval(bysub, parent.frame(), parent.frame()) : 
  object 'XC' not found
Calls: cellBC -> [ -> [.data.table -> eval -> eval
In addition: Warning message:
In data.table::fread(bccount_file, header = FALSE, col.names = c("XC",  :
  File '/public/groups/hausslerlab/people/kivilcim/iMGL/outputall/KDiMGLall.BCstats.txt' has size 0. Returning a NULL data.table.
Execution halted
"
This did not happen when I was using the same data 2 days ago, and  my yaml file appears to be formatted correctly.  I have seen some comments about this issue in the zUMIs repository. Let me know if you believe it is a better idea to post it there, here or if I should start a new issue. Depending on your answer I can include all my files,  links to what I have tried and explain the situation better. Truly appreciative of your help! Thank you so much!
